src/Castle.Windsor.Extensions.DependencyInjection/Scope/ExtensionContainerScopeBase.cs

```diff
@@ -0,0 +1,66 @@
+// Copyright 2004-2020 Castle Project - http://www.castleproject.org/
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+namespace Castle.Windsor.Extensions.DependencyInjection.Scope
+{
+	using System;
+	
+	using Castle.Core;
+	using Castle.MicroKernel;
+	using Castle.MicroKernel.Lifestyle.Scoped;
+
+	internal  abstract class ExtensionContainerScopeBase : ILifetimeScope
+	{
+		public static string TransientMarker = "Transient";
```

Comment:
This constant should be marked `readonly` or defined as a constant.

Reply:
done